src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -268,23 +276,37 @@ def _get_bicep_installed_version(bicep_executable_path):
     return _extract_version(installed_version_output)
 
 
-def _get_bicep_download_url(system, release_tag, target_platform=None):
-    download_url = f"https://downloads.bicep.azure.com/{release_tag}/{{}}"
-
-    if target_platform:
-        executable_name = "bicep-win-x64.exe" if target_platform == "win-x64" else f"bicep-{target_platform}"
-        return download_url.format(executable_name)
+def _has_musl_binaries():
```

Comment:
nit: Might make sense to use the name `_has_musl_library_only` to indicate that this function checks for the presence of the musl library while explicitly excluding x86_64.

Reply:
Fixed.